sites/evaluationengineering/server/templates/magazine/index.marko

```diff
@@ -9,7 +9,7 @@ $ const description = site.get('magazines.description');
   <@above-container>
     <endeavor-gam-ad-unit-out-of-page name="wa" />
     <endeavor-gam-ad-unit-out-of-page name="reskin" />
-    <endeavor-gam-ad-unit-define-display name="BS" modifiers=["top-of-page"] />
+    <endeavor-gam-ad-unit-define-display name="lb1" modifiers=["top-of-page"] />
```

Comment:
With the recent changes for the reskin ad, can you move the leaderboards right after above container? They should be placed between the closing above container but before the page wrapper div

Reply:
@zarathustra323 - Done.